### PR TITLE
Add rp2040 boards

### PR DESCRIPTION
**Pull request by Jpe230**
## Description

Add commands to send images of the pinout for the Adafruit KB2040 and the Sparkfun Pro Micro RP2040

### CODE REVIEW

**Comment by tzarc** [comment]
Should we perhaps make this `sparkfun2040` and `adafruit2040`? Should be easier to remember?

File: qmkbot.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ require('dotenv').config();
 const token = process.env.TOKEN;
 
 // Import utils.js
-const {prefix, baseurl, msg, docsSwitch, authroles, parse, bare, firmware, toolbox, plainhelp, disclaimer, ohshitgit, git, xkcd, promicro, protonc, elitec, blackpill, bluepill, msys, coc, kbdfans, lighting, wsl, vid, vidq, automark, markdown, checkrole, sonixinvite, sonix, openrgbinvite, openrgb, vialinvite, vial, iconfrominvite, inputlanguage, standards, snip, zmk, zmkinvite, prstatus} = require('./utils.js');
+const {prefix, baseurl, msg, docsSwitch, authroles, parse, bare, firmware, toolbox, plainhelp, disclaimer, ohshitgit, git, xkcd, promicro, protonc, elitec, blackpill, bluepill, sparkfunrp, adafruitkb, msys, coc, kbdfans, lighting, wsl, vid, vidq, automark, markdown, checkrole, sonixinvite, sonix, openrgbinvite, openrgb, vialinvite, vial, iconfrominvite, inputlanguage, standards, snip, zmk, zmkinvite, prstatus} = require('./utils.js');
```
Comment (yanfali) [inline qmkbot.js]: at some point it just gets easier to stop doing this kind of import and just using utils.<something>

**Review by yanfali — APPROVED** [review]

**Comment by Jpe230** [comment]
> Should we perhaps make this `sparkfun2040` and `adafruit2040`? Should be easier to remember?

The parser borks if there are numbers in the command :( that's why the commands don't use *2040

**Comment by yanfali** [comment]
> > Should we perhaps make this `sparkfun2040` and `adafruit2040`? Should be easier to remember?
> 
> The parser borks if there are numbers in the command :( that's why the commands don't use *2040

would pico be better compromise?

**Comment by Jpe230** [comment]
> > > Should we perhaps make this `sparkfun2040` and `adafruit2040`? Should be easier to remember?
> > 
> > 
> > The parser borks if there are numbers in the command :( that's why the commands don't use *2040
> 
> would pico be better compromise?

Pico refers to a board from the Pi Foundation, while it uses the same MCU, it isnt part of this PR.

**Comment by Jpe230** [comment]
Rebased my PR and updated the REGEX to allow the use of numbers.

The commands now are `sparkfun2040` and `adafruit2040`

I tested that all current commands still works with the modified REGEX

**Pull request closed without merging**